docs/components/quick-links.vue

```diff
@@ -0,0 +1,103 @@
+<template>
+  <nav :class="['bd-quick-links', 'mb-3', { 'd-none': quickLinksMoved }]">
+    <header>
+      <b-button
+        v-b-toggle.bd-quick-links-collapse
+        class="font-weight-bold"
+        variant="outline-secondary"
+        size="sm"
+        block
+      >
+        <span v-if="quickLinksVisible">Hide</span>
+        <span v-else>Show</span>
+        Page table of contents
+      </b-button>
+    </header>
+    <b-collapse id="bd-quick-links-collapse" v-model="quickLinksVisible" tag="ul">
+      <li v-for="h2 in toc.toc" :key="h2.href">
+        <b-link :href="h2.href" @click="scrollIntoView($event, h2.href)">
+          <span v-html="h2.label"></span>
+        </b-link>
+      </li>
+    </b-collapse>
+  </nav>
+</template>
+
+<style scoped lang="scss">
+#bd-quick-links-collapse {
+  border-left: 5px solid #ccc;
+  padding-left: 2.5rem;
+  margin-top: 1rem;
+}
+</style>
+
+<script>
+import { makeTOC, offsetTop, scrollTo } from '~/utils'
+
+export default {
+  name: 'BDVToc',
+  data() {
+    return {
+      readme: '',
+      meta: null,
+      offset: 0,
+      quickLinksVisible: false
+      quickLinksMoved: false
+    }
+  },
+  computed: {
+    toc() {
+      return makeTOC(this.readme, this.meta)
+    }
+  },
+  created() {
+    this.$root.$on('setTOC', (readme, meta) => {
+      this.readme = readme
+      this.meta = meta || null
+    })
+  },
+  mounted() {
+    const $body = document.body
+    // Set the correct offset based on the header height
+    const $header = $body.querySelector('header.navbar')
+    if ($header) {
+      this.offset = $header.offsetHeight + 6
+    }
+    // Move the quick links to the correct position, if possible
+    const $referenceNode = $body.querySelector('.bd-lead') || $body.querySelector('h1')
+    if ($referenceNode) {
+      // IE 11 doesn't support the .after() method, and appears
+      // that the polyfill doesn't polyfill this method
+      // $referenceNode.after(this.$el)
+      $referenceNode.insertAdjacentElement('afterend', this.$el)
```

Comment:
@jackmu95 had to switch this to `.insertAdjacentElement(...)` since it appears the polyfill doesn't polyfill `Element.prototype.after()` in IE